abc.go

```diff
@@ -131,9 +131,9 @@ func NewColoredLogger(wrapped WriterLogger) WriterLogger {
 	}
 }
 
-// Must panics, if the given error is not nil.
+// panics, if the given error is not nil.
```

Comment:
Should be
```go
// must panics, if the given error is not nil.
```

Reply:
for unexportend functions, you don't need to name the function itself, and the comment is for internal purposes only.

Since it's already a comment on the function, how about

`// this function panics, if the given error is not nil.`

Reply:
https://blog.golang.org/godoc-documenting-go-code states that:
> Notice this comment is a complete sentence that begins with the name of the element it describes.

There is no warning generated by go vet or another go lint tool, since the function is private, but that is not godoc convention.